<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Termometro</class>
 <widget class="QWidget" name="Termometro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Termometro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Downloads/termometro (1).png</normaloff>:/Downloads/termometro (1).png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="grpC">
     <property name="title">
      <string>Centigrados</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdC">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialC">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-18</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpF">
     <property name="title">
      <string>Farenheit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdF">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialF">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>210</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpK">
     <property name="title">
      <string>Kelvin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdK">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialK">
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="minimum">
         <number>255</number>
        </property>
        <property name="maximum">
         <number>372</number>
        </property>
        <property name="value">
         <number>273</number>
        </property>
        <property name="sliderPosition">
         <number>273</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdC</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialF</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdF</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialK</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdK</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
